Resources/Prototypes/Entities/Objects/Tools/lantern.yml

```diff
@@ -42,3 +42,11 @@
       radius: 5
       energy: 10
       color: "#FFC458"
+    - type: Flash
```

Comment:
Additionally, in order for the sprite to work, it needs to be given states for "flashing" and "burnt". You could probably just reuse existing sprites for those, as i don't think it matters too much. As it stands right now, it errors when it flashes and also displays an error sprite upon being burnt out.